src/plugins/template/backend/ops/group_normalization.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+// clang-format off
+#include "evaluate_node.hpp"
+
+#include "ngraph/runtime/reference/group_normalization.hpp"
+
+#include "openvino/op/group_normalization.hpp"
+// clang-format on
+
+using namespace ov;
+
+template <element::Type_t DATA_ET>
+bool evaluate(const std::shared_ptr<ov::op::v12::GroupNormalization>& node,
+              const ov::HostTensorVector& outputs,
+              const ov::HostTensorVector& inputs) {
```

Comment:
It's worth to set shape at the output tensor (in case of dynamic shapes)
```suggestion
              const ov::HostTensorVector& inputs) {
              outputs[0]->set_shape(inputs[0]->get_shape());
```
or even call the shape_infer function:
https://github.com/openvinotoolkit/openvino/blob/fc9e0ce315e77d8235b9c45917bf5cde125feecd/src/core/shape_inference/include/group_normalization_shape_inference.hpp#L13

Reply:
Done.